rskj-core/src/test/java/co/rsk/signing/KeyFileCheckerTest.java

```diff
@@ -0,0 +1,88 @@
+package co.rsk.signing;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.internal.util.collections.Sets;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.attribute.PosixFilePermission;
+
+/**
+ * Created by ajlopez on 29/12/2016.
+ */
+public class KeyFileCheckerTest {
+    private static final String KEY_FILE_PATH = "./keyfiletest.txt";

```

Comment:
I don't think we should be writing to disk for testing, but I can't see a simple alternative. We might end up mocking lots of IO-related classes.

Reply:
Legacy code. Agree on not writing to disk in tests. That said, this is probably not very relevant at the moment and can do the corresponding refactor at some point. Unless, of course, you have a better solution.

Reply:
Could we have an issue in our internal issue tracker, describing this pending task?